public/app/features/explore/LiveTailButton.tsx

```diff
@@ -53,16 +55,46 @@ const getStyles = memoizeOne((theme: GrafanaTheme) => {
       }
       @keyframes pulse {
         0% {
-          color: ${textColor};
+          color: ${pulseTextColor};
+          text-shadow: 0 0 0 ${orange};
         }
         50% {
           color: ${orange};
+          text-shadow: 0 0 4px
+            ${tinycolor(orange)
+              .setAlpha(0.7)
+              .toString()};
         }
         100% {
-          color: ${textColor};
+          color: ${pulseTextColor};
+          text-shadow: 0 0 0 ${orange};
         }
       }
     `,
+    stopButtonEnter: css`
+      label: stopButtonEnter;
+      width: 0;
+      opacity: 0;
+      overflow: hidden;
+    `,
+    stopButtonEnterActive: css`
+      label: stopButtonEnterActive;
+      opacity: 1;
+      width: 32px;
+      transition: opacity 500ms, width 500ms;
+    `,
+    stopButtonExit: css`
+      label: stopButtonExit;
+      width: 32px;
```

Comment:
Animating to non explicit width does not work so this is bit brittle but not sure how to do it other way.